### PR TITLE
Exclude unused bip39 wordlists from dist bundle

### PR DESCRIPTION
**Pull request by zone117x**
Reduce blockstack.js dist bundle size https://github.com/blockstack/blockstack.js/issues/582

Exclude unused bip39 wordlists from dist bundle. Removes around 240KB (uncompressed) of data. 

image

Acceptance criteria:
* Ensure the `dist/blockstack.js` bundle still works normally in web apps -- that is the only file this PR changes.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -78,7 +79,17 @@ module.exports = (env, argv) => {
       libraryTarget: 'umd',
       globalObject: 'this'
     },
-    plugins: []
+    plugins: [
+      // Can't get this regex exclusion to work...
+      // new webpack.IgnorePlugin(/\.\/wordlists\/(?!.*english.*)\.json/)
```
Comment (zone117x) [inline webpack.config.js]: I couldn't get this regex to work so just listed out all non-english files. LMK if you know how to fix this

**Comment by zone117x** [comment]
TODO: Test out another regex example for excluding all but english worldlists

**Comment by zone117x** [comment]
@yknl Updated the PR with the single regex filter you suggested -- it works. Thanks!

**Review by yknl — APPROVED** [review]
LGTM

**Comment by zone117x** [comment]
Going to merge this since this only changes the dist bundle.

**Pull request merged**